fvm/evm/emulator/config.go

```diff
@@ -17,6 +17,12 @@ var (
 	bigZero = big.NewInt(0)
 )
 
+var (
+	PreviewnetPragueActivation = uint64(0)          // already on Prague for PreviewNet
+	TestnetPragueActivation    = uint64(1746039600) // Wednesday, April 30, 2025 7:00:00 PM GMT
```

Comment:
TODO(m-Peter): Update this once we set the proper time for the Pectra upgrade on Testnet.

Reply:
nit: I would put them far into the future until we know the exact time. So that if this accidentally merged we wont have problems.